sympy/codegen/pynodes.py

```diff
@@ -1,4 +1,11 @@
 from .abstract_nodes import List as AbstractList
+from .ast import Token
+
 
 class List(AbstractList):
     pass
+
+
+class NumExprEvaluate(Token):
+    """represents a call to `numexpr`s `evaluate`"""
```

Comment:
In the SymPy sphinx default, backtick will lead to LaTeX generation. I think you need something like
```suggestion
    """represents a call to :class:`numexpr`s :func:`evaluate`"""
```
to make it work properly.